OpenIDConnectClient.php

```diff
@@ -417,7 +417,7 @@ public function getRedirectURL() {
               ?: @$_SERVER['SERVER_ADDR'];
 
         // Don't include port if it's 80 or 443 and the protocol matches
-        $port = (443 == $port) || (80 == $port) ? '' : ':' . $port;
+        $port = ($protocol === 'https' && $port === 443) || ($protocol === 'http' && $port === 80) ? '' : ':' . $port;
```

Comment:
It looks like it's behind one commit. I committed a fix to master a few days ago. No need to fiddle with git commands. Just change this one line and you should be set. 👍 

https://github.com/jumbojett/OpenID-Connect-PHP/commit/fcdf8635a60d31460700336e1abd905762bf41c9